tests/UlidTest.php

```diff
@@ -64,16 +79,38 @@ public function testGeneratesLexographicallySortableUlids(): void
 
     public function testCreatesFromUppercaseString(): void
     {
-        $this->assertEquals('01AN4Z07BY79KA1307SR9X4MV3', (string) Ulid::fromString('01AN4Z07BY79KA1307SR9X4MV3'));
-        $this->assertEquals('01AN4Z07BY79KA1307SR9X4MV3', (string) Ulid::fromString('01AN4Z07BY79KA1307SR9X4MV3', false));
-        $this->assertEquals('01an4z07by79ka1307sr9x4mv3', (string) Ulid::fromString('01AN4Z07BY79KA1307SR9X4MV3', true));
+        $ulid_default = Ulid::fromString('01AN4Z07BY79KA1307SR9X4MV3');
+        $this->assertEquals('01AN4Z07BY79KA1307SR9X4MV3', (string) $ulid_default);
+        $this->assertEquals('01AN4Z07BY', $ulid_default->getTime());
+        $this->assertEquals('79KA1307SR9X4MV3', $ulid_default->getRandomness());
+
+        $ulid_no_lowercase = Ulid::fromString('01AN4Z07BY79KA1307SR9X4MV3', false);
```

Comment:
I don't really like these snake case variables. Better would be to name them `$uppercase`, `$lowercase` and `$default`.